src/main/java/bookstore/entity/Role.java

```diff
@@ -0,0 +1,35 @@
+package bookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false, unique = true)
+    @Enumerated(EnumType.STRING)
+    private RoleName name;
+
+    @Override
+    public String getAuthority() {
+        return name.name();
```

Comment:
as for me its better one time return 
```suggestion
        return "ROLE_" + name.name();
```

And make enum without this prefix, but its up to you